config/environments/development.rb

```diff
@@ -38,4 +38,39 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+
+
+  config.action_mailer.default_url_options = {:host => 'localhost:3000'}
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.smtp_settings = {
+    :address   => "smtp.mandrillapp.com",
+    :port      => 587, # ports 587 and 2525 are also supported with STARTTLS
+    :enable_starttls_auto => true, # detects and uses STARTTLS
+    :user_name => "xpp@me.com",
+    :password  => "hj1ybw_z9LxpxxKHDggwgQ", # SMTP password is any valid API key
```

Comment:
Never include your passwords in git. Always use environment variables from the beginning.